pkg/cloud-provider/cloudapi/azure/azure_security.go

```diff
@@ -297,9 +297,14 @@ func (computeCfg *computeServiceConfig) buildEffectiveNSGSecurityRulesToApply(ap
 			continue
 		}
 		// skip any rules not created by nephe.
```

Comment:
nephe rules will be created from ruleStartPriority.

Reply:
Done